contracts/handlers/ERC20Handler.sol

```diff
@@ -101,11 +95,7 @@ contract ERC20Handler is IDepositExecute, HandlerHelpers, ERC20Safe {
         address tokenAddress = _resourceIDToTokenContractAddress[resourceID];
         require(_contractWhitelist[tokenAddress], "provided tokenAddress is not whitelisted");
 
-        if (_burnList[tokenAddress]) {
-            burnERC20(tokenAddress, depositer, amount);
-        } else {
-            lockERC20(tokenAddress, depositer, address(this), amount);
-        }
+        lockERC20(tokenAddress, depositer, address(this), amount);
```

Comment:
Update the method description in line 79.

Reply:
Done